dev/github_jira_sync.py

```diff
@@ -77,8 +77,8 @@ def get_jira_prs():
                 result = result + [(jira, pull)]
 
         # Check if there is another page
-        link_header = filter(lambda k: k.startswith("Link"), page.info().headers)[0]
-        if "next" not in link_header:
+        link_headers = filter(lambda k: k.startswith("Link"), page.info().headers)
```

Comment:
Just wanted to leave a note that `filter` returns a generator from Python 3... in fact this whole script should be fixed anyway as of SPARK-27889 separately.

```python
>>> filter(lambda x: x, ['a'])[0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'filter' object is not subscriptable
>>> list(filter(lambda x: x, ['a']))[0]
'a'
```